blivet/devicefactory.py

```diff
@@ -332,7 +332,7 @@ def __init__(self, storage, **kwargs):
             setattr(self, setting, value)
 
         self.fstype = None  # not included in default_settings b/c of special handling below
-        self.min_luks_entropy = kwargs.get("min_luks_entropy") or luks_data.min_entropy
+        self.min_luks_entropy = kwargs.get("min_luks_entropy", luks_data.min_entropy)
```

Comment:
The old code here is actively protecting against using a passed-in `0` or `None`. Is that not desirable?

Reply:
Yeah, `None` should be replaced  by a number, I think. `0` should, however, be respected.